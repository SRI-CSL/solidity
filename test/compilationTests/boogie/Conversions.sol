pragma solidity ^0.4.23;

contract SomeContract {
    uint public stateVar;

    function someFunc() public view returns (uint) {
        return stateVar;
    }
}

contract OtherContract {
    function otherFunc(address addr) public view returns (uint) {
        return SomeContract(addr).someFunc();
    }
}