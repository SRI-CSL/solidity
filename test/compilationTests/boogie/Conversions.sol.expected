Verifying SomeContract::someFunc: OK
Verifying OtherContract::otherFunc: OK
Verifying Conversions::someFunc: OK
No errors found.
